static/js/components/ProfileImageUploader.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import R from 'ramda';
+import Dropzone from 'react-dropzone';
+import Cropper from 'react-cropper';
+import Button from 'react-mdl/lib/Button';
+import type { ImageUploadState } from '../reducers/image_upload';
+
+const onDrop = R.curry((startPhotoEdit, files) => startPhotoEdit(...files));
+
+const dropZone = onDrop => (
+  <Dropzone
+    onDrop={onDrop}
+    className="photo-active-item photo-dropzone"
+    activeClassName="photo-active-item photo-dropzone active"
+  >
+    <div>
+      Drag a photo here or click to select a photo.
+    </div>
+  </Dropzone>
+);
+
+class CropperWrapper extends React.Component {
```

Comment:
sure, I was hoping to avoid using any class declarations but for this I needed a `ref` to grab the cropped data from.